packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -133,3 +153,40 @@ export function hangupCall(call) {
     });
   };
 }
+
+export function listeningForIncomingCalls() {
```

Comment:
Should this be `listenForIncomingCalls` to make it consistent with our other action names/